### DISCUSSION TITLE
请问这个支持所有订阅都用同一套规则吗

### DISCUSSION THREAD

**Discussion by yidasanqian** [clash-verge-rev/clash-verge-rev#2026] [Q&A]
比如我有三个订阅源，我想它们都使用一套代理规则，支持配置吗，有没有教程？

**Comment by MC-dusk** [comment]
https://github.com/clash-verge-rev/clash-verge-rev/issues/1014
https://github.com/clash-verge-rev/clash-verge-rev/issues/824

这个页面底下有个示例：https://www.clashverge.dev/guide/script.html

**Reply by MC-dusk** [reply]
如果有人想直接cv，可以用下面几个链接：
https://github.com/dahaha-365/YaNet/blob/main/Mihomo/global_script.js
https://github.com/Kiritocyz/Clash
https://github.com/mihomo-party-org/override-hub/blob/main/javascript/%E5%B8%83%E4%B8%81%E7%8B%97%E7%9A%84%E8%AE%A2%E9%98%85%E8%BD%AC%E6%8D%A2.js

**Comment by MC-dusk** [comment]
不过话说回来，还是希望能直接支持cfw的预处理脚本，网上有很多现成的，如果能直接用更好，现在需要改成verge的处理模式，还得yaml和js还得分开写

**Reply by iwalke** [reply]
预处理也有的
![image](https://github.com/user-attachments/assets/409a804b-60ee-4d05-b7e1-0918ac78855d)

[script](https://www.clashverge.dev/guide/script.html#1) 比 cfw 的预处理更强大，超喜欢这个功能，直接当订阅转换用。就像 app 把订阅的【代理】字段抽出来一样，然后自己定制规则。

**Reply by MC-dusk** [reply]
你没懂意思，cfw的预处理是yaml、command和js混合写法的，verge是另一套体系，不能直接复制粘贴
对于新手yaml会简单些，而js则是通用的，如果是纯js写法的话，粘贴到哪都行
我也是从cfw迁移过来，没办法用js重写了一遍

**Comment by yidasanqian** [comment]
谢谢回复，看起来配置有点麻烦，做的太复杂了
